docs/chapters/baseplatform/building-PELUX-sources.rst

```diff
@@ -22,76 +37,96 @@ image eventually, as the project evolves further and further away from vanilla
-    <style> .available {color:green} .unavailable {color:red} </style>
-
-+--------------------------------------------+------------------+------------------+------------------+----------------+
-|                                            |      Variant name                                                       |
-+          Image name                        +------------------+------------------+------------------+----------------+
-|                                            | intel            | intel-qtauto     | rpi              | rpi-qtauto     |
-+============================================+==================+==================+==================+================+
-| core-image-pelux-minimal                   | :available:`✔`   | :available:`✔`   | :available:`✔`   | :available:`✔` |
-+--------------------------------------------+------------------+------------------+------------------+----------------+
-| core-image-pelux-minimal-dev               | :available:`✔`   | :available:`✔`   | :available:`✔`   | :available:`✔` |
-+--------------------------------------------+------------------+------------------+------------------+----------------+
-| core-image-pelux-qtauto-neptune            | :unavailable:`✘` | :available:`✔`   | :unavailable:`✘` | :available:`✔` |
-+--------------------------------------------+------------------+------------------+------------------+----------------+
-| core-image-pelux-qtauto-neptune-dev        | :unavailable:`✘` | :available:`✔`   | :unavailable:`✘` | :available:`✔` |
-+--------------------------------------------+------------------+------------------+------------------+----------------+
-| core-image-pelux-minimal-update            | :available:`✔`   | :available:`✔`   | :available:`✔`   | :available:`✔` |
-+--------------------------------------------+------------------+------------------+------------------+----------------+
-| core-image-pelux-minimal-dev-update        | :available:`✔`   | :available:`✔`   | :available:`✔`   | :available:`✔` |
-+--------------------------------------------+------------------+------------------+------------------+----------------+
-| core-image-pelux-qtauto-neptune-update     | :unavailable:`✘` | :available:`✔`   | :unavailable:`✘` | :available:`✔` |
-+--------------------------------------------+------------------+------------------+------------------+----------------+
-| core-image-pelux-qtauto-neptune-dev-update | :unavailable:`✘` | :available:`✔`   | :unavailable:`✘` | :available:`✔` |
-+--------------------------------------------+------------------+------------------+------------------+----------------+
-
-When done fetching the sources, create a build directory and set up bitbake.
-``TEMPLATECONF`` tells the ``oe-init-build-env`` script which path to fetch
-configuration samples from.
+Once you have synced the repo, run the following to configure the environment for Qt Automotive + Neptune 3 UI images. 
```

Comment:
Remove text '+ Neptune 3 UI"

(enough in my opinion to mention neptune only in the context of specifying supported images)